hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/KeyValueContainer.java

```diff
@@ -260,7 +260,8 @@ private void writeToContainerFile(File containerFile, boolean isCreate)
     } catch (IOException ex) {
       onFailure(containerData.getVolume());
       throw new StorageContainerException("Error while creating/ updating " +
-          ".container file. ContainerID: " + containerId, ex,
+          ".container file. ContainerID: " + containerId + " Container path: " +
```

Comment:
```suggestion
          " container file. ContainerID: " + containerId + ", container path: " +
```

Reply:
I think the `.` was originally there to signify the file extension of the file is `.container`. I don't think it matters either way, just a note.

Reply:
Thanks for spotting this, I've fixed it.